.changeset/remove-default-server-node-polyfills.md

```diff
@@ -0,0 +1,69 @@
+---
+"@remix-run/dev": major
+---
+
+Remove default Node.js polyfills from the server build when targeting non-Node.js platforms.
+
+Any Node.js polyfills that are required for your server code to run on non-Node.js platforms must be manually specified in `remix.config.js` using the `serverNodeBuiltinsPolyfill` option.
+
+```js
+exports.serverNodeBuiltinsPolyfill = {
+  modules: {
+    path: true, // Provide a JSPM polyfill
+    fs: "empty", // Provide an empty polyfill
+  },
+};
+```
+
+For reference, the complete set of default polyfills from Remix v1 can be manually specified as follows:
```

Comment:
This feels potentially big for release notes - is there any benefit to putting this in the docs - maybe collapsed in a summary/details section?

Reply:
It's in the v2 guide so we could just leave it there for now. I'll drop this from the release notes.